TextToSpeech/impl/TTSSpeaker.cpp

```diff
@@ -777,6 +782,7 @@ void TTSSpeaker::GStreamerBusWatchThreadFunc(void *ctx) {
                 speaker->handleMessage(msg);
```

Comment:
This "TTSSpeaker::GStreamerBusWatchThreadFunc" looks to be a tight loop, will this be waiting for event or will be polling for events continuously?

Reply:
Added the Gmain loop. So that we can get the GST events from bus watch concept.  So it is not polling for events anymore.